core/arch/aarch64/codec.txt

```diff
@@ -756,7 +756,7 @@ x0110111xxxxxxxxxxxxxxxxxxxxxxxx  tbnz   tbz
 0x001100000000000010xxxxxxxxxxxx  st1    memvm : vmsz vt0 vt1 vt2 vt3
 0x001100000000000100xxxxxxxxxxxx  st3    memvm : vmsz vt0 vt1 vt2
 0x001100000000000110xxxxxxxxxxxx  st1    memvm : vmsz vt0 vt1 vt2
-0x001100000000000111xxxxxxxxxxxx  st1    memvm : vmsz vt0
+0x001100000000000111xxxxxxxxxxxx  st1    memvm : vt0 vmsz
```

Comment:
Did we flip the position of the vmsz operand here to be more in line with the XXX_sz operands, which come last?

Reply:
Yes

Reply:
Makes sense! Is there a reason you did not change it for ld1?

Reply:
> Makes sense! Is there a reason you did not change it for ld1?

ld1 doesn't need to change as its fields are already correctly positioned w.r.t. encoding string:
```
0x001100010000000111xxxxxxxxxxxx  ld1    vt0 : memvm vmsz
```

Reply:
Right!